suite.go

```diff
@@ -0,0 +1,110 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import (
+	"reflect"
+	"strings"
+	"unicode"
+	"unicode/utf8"
+)
+
+// RunSuite runs all the tests defined in the given suite as a separate
```

Comment:
s/all the tests defined in the given suite/each test method defined on the given value/

?